tools/portlet_spring_mvc_tmpl/docroot/WEB-INF/jsp/view.jsp

```diff
@@ -0,0 +1,8 @@
+<%@page contentType="text/html" pageEncoding="UTF-8"%>
+
+<%@include file="../init.jspf"%>
+
+
```

Comment:
But it would be good to show the release info since the controller has prepared that for the view.